### ISSUE TITLE
AVT 1 - Single line, Multi line CodeSnippet, and CopyButton has DAP violation

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3734]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP IBM Accessibility July 2019 Ruleset

## Detailed Description
Go to Single line React CodeSnippet
Run DAP and the following issue is found on the copy button (see screenshot):
<img width="1255" alt="Screen Shot 2019-08-14 at 4 14 23 PM" src="https://user-images.githubusercontent.com/21676914/63057299-8b789300-beaf-11e9-8991-f521803df541.png">
Note: The button is missing an aria-label.

**Comment by snidersd** [comment]
Ran DAP on CopyButton component that is used in the CodeSnippet and verified it has the same issue. See screenshot below:
<img width="1327" alt="Screen Shot 2019-10-02 at 2 09 55 PM" src="https://user-images.githubusercontent.com/21676914/66074383-b92ca080-e51e-11e9-9038-261a5a8f798a.png">
